index.bs

```diff
@@ -10135,7 +10186,7 @@ Note: The HTML Standard defines how a security check is performed. [[!HTML]]
 
 For every non-callback [=interface=] that is [=exposed=] in
 a given ECMAScript global environment and that is not declared with
-the [{{NoInterfaceObject}}] [=extended attribute=],
+the [{{NoInterfaceObject}}] or [{{LegacyNamespace}}] [=extended attribute|extended attributes=],
```

Comment:
s/[=extended attribute|extended attributes=]/[=extended attributes=]/ (Bikeshed should recognize the plurals.